app/models/solidus_subscriptions/subscription.rb

```diff
@@ -241,6 +241,13 @@ def billing_address_to_use
       billing_address || user.bill_address
     end
 
+    def clear_past_installments
```

Comment:
Is there a reason for this method to live in Subscription rather than simply putting the code in Processor? I'm afraid someone might think this is a public API, which it isn't.

Reply:
It makes sense. Moving it inside.